graylog2-web-interface/src/views/components/aggregationbuilder/SeriesSelect.tsx

```diff
@@ -168,10 +168,9 @@ class SeriesSelect extends React.Component<Props, State> {
               options={options}
               value={series.map(_wrapOption)}
               components={_components}
-              onClose={this._onClose}
               closeMenuOnSelect={false}
+              onMenuClose={this._onClose}
               menuShouldScrollIntoView
-              tabSelectsValue
```

Comment:
Is this removed on purpose?

Reply:
I removed this prop, because we always set it to `false` in the views `Select`. I tested this case with `tabSelectsValue` and prefer to keep this disabled.